### PR TITLE
docs: how to configure a custom resource

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan,

Please review the following commits I made in branch nicks/frameworks:

c70c6e08e7a5982d5e193e72d9a98268bee75e1f (2020-07-02 19:12:06 -0400)
docs: how to configure a custom resource

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: docs/custom_resource.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+But your team may define your own custom resources! Or you may use a project
+like KubeDB that uses custom resources to operate a database.
+
+Tilt needs your help to recognize them.
+
+## Custom Resource Functions
+
+For any resource, Tilt needs to know 3 things:
+
+### Is this an indepdent resource?
```
Comment (jazzdan) [inline docs/custom_resource.md:22]:
```suggestion
### Is this an independent resource?
```

But also, what happens if it's not an independent resource, what should you do then?

(I know the answer, I just think that the way that this is phrased leaves that question unanswered)

File: docs/custom_resource.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+```
+k8s_resource(new_name='postgres',
+             extra_pod_selectors=[{'kubedb.com/name': 'quick-postgres'}])
+```
+
+See the [k8s_resource API docs](api.html#k8s_resource) for more detail, or look
+at [this
+example](https://github.com/tilt-dev/tilt-example-frameworks/blob/master/kubedb-postgres/Tiltfile)
+that selects the Postgres pods created by the KubeDB postgres operator.
```
Comment (jazzdan) [inline docs/custom_resource.md:66]:
```suggestion
that selects the Postgres pods created by the KubeDB Postgres operator.
```

(We should just be consistent, doesn't matter to me how we capitalize it)

File: docs/custom_resource.md (hunk 1 above)
Comment (nicks) [reply docs/custom_resource.md:22]: done

File: docs/custom_resource.md (hunk 2 above)
Comment (nicks) [reply docs/custom_resource.md]: done

**Pull request merged**